Subject: This patch performs Debian specific customizations of
 setup.py. It removes script and package data installation and uses the
 system wide setuptools installation, which is actually distribute on
 Debian systems.
Author: Jan Dittberner <jandd@debian.org>
--- a/setup.py
+++ b/setup.py
@@ -3,10 +3,6 @@
 PROJECT = 'virtualenvwrapper'
 VERSION = '2.1'
 
-# Bootstrap installation of Distribute
-import distribute_setup
-distribute_setup.use_setuptools()
-
 from setuptools import setup, find_packages
 
 from distutils.util import convert_path
@@ -132,9 +128,6 @@
 
     platforms = ('Any',),
 
-    scripts = ['virtualenvwrapper.sh', 
-               ],
-
     provides=['virtualenvwrapper',
               'virtualenvwrapper.user_scripts',
               ],
@@ -142,14 +135,7 @@
 
     namespace_packages = [ 'virtualenvwrapper' ],
     packages = find_packages(),
-    include_package_data = True,
-    # Scan the input for package information
-    # to grab any data files (text, images, etc.) 
-    # associated with sub-packages.
-    package_data = find_package_data(PROJECT, 
-                                     package=PROJECT,
-                                     only_in_packages=False,
-                                     ),
+    include_package_data = False,
 
     entry_points = {
         #'console_scripts': [ 'venvw_hook = virtualenvwrapper.hook_loader:main' ],
